public/menu.js

```diff
@@ -23,6 +41,20 @@ function setMainMenu() {
     {
       label: 'Edit',
       submenu: [
+        {
+          label: 'Lower Opacity',
+          accelerator: 'CmdOrCtrl+-',
```

Comment:
Change this shortcut to `CmdOrCtrl+Down`

Reply:
I agree. `CmdOrCtrl+=` and `CmdOrCtrl+-` can be interpreted as zoom shortcuts.